ipi/engine/motion/instanton.py

```diff
@@ -938,6 +959,39 @@ def pre_step(self, step=None, adaptative=False):
         self.qtime = -time.time()
         info("\n Instanton optimization STEP {}".format(step), verbosity.low)
 
+        # Choosing the forcefield that has returned friction tensor in extras
```

Comment:
The current PR is about the extras communication, I would delete this here.

Reply:
That is true, however, I think that the point why we are doing the whole modification of the extras input, is to have access to the friction. So I included this part to show how one accesses, the friction how to transfer that data structure into a friction object to work with.

Reply:
I agree that this can/should be deleted.

Reply:
I understand, yes :) I removed it.

Reply:
is this resolved eszter? if so please press "resolve conversation"